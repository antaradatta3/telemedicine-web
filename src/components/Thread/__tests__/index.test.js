import React from 'react';
import { mount } from 'enzyme';

import Thread from '../index';

describe('Component: Thread', () => {
  const props = {};

  it('Thread matches snapshot', () => {
    const component = mount(<Thread {...props} />);
    expect(component).toMatchSnapshot();
  });
});
