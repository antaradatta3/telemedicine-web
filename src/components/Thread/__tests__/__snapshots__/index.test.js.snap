// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Thread Thread matches snapshot 1`] = `
<Thread>
  <div />
</Thread>
`;
